### PR TITLE
Update 7.x-2.x README for REST API and associated links.

### PR DESCRIPTION
**Pull request by mccrodp**
As mentioned in https://github.com/MarketoMA/marketo_ma/issues/13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,54 +61,55 @@ Multiple options are available for how captured data is submitted to Marketo.
...
 
 The fields configured here will be available for mapping to User and Webform fields.
 They should match those that are defined in your Marketo admin under
 Admin > Field Management. Additional information regarding Marketo fields can be
-found in the Marketo articles [Field Management](http://community.marketo.com/MarketoDeepDive?id=kA5500000008RWQCA2)
-and [Export a List of All Marketo API Field Names](http://community.marketo.com/MarketoArticle?id=kA050000000KytHCAS).
+found in the Marketo articles [Field Management](http://docs.marketo.com/display/public/DOCS/Field+Management)
+and [Export a List of All Marketo API Field Names](http://docs.marketo.com/display/public/DOCS/Export+a+List+of+All+Marketo+API+Field+Names).
 
 **Marketo Fields**
```
Comment (jyokum) [inline README.md]: Field management has changed and needs to be reflected in docs.

**Review by jyokum — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mccrodp) [reply README.md]: 👍 Will add now

Comment (mccrodp) [reply README.md]: Done. Please review @jyokum!

File: README.md [hunk 2]
```diff
@@ -61,54 +61,59 @@ Multiple options are available for how captured data is submitted to Marketo.
...
+"Retrieve from Marketo". By default, all fields retrieved will be available for 
+mapping to Webform and User Profile fields. It is possible to limit the 
+available fields by selecting them via the Field Definition form table. 
+Read-only fields are displayed but are never available for mapping. See all 
+fields that are defined in your Marketo admin under 
+Admin > Database Management > Field Management. Additional information regarding 
+Marketo fields can be found in the Marketo articles [Field Management](http://docs.marketo.com/display/public/DOCS/Field+Management)
+and [Export a List of All Marketo API Field Names](http://docs.marketo.com/display/public/DOCS/Export+a+List+of+All+Marketo+API+Field+Names).
 
 **Marketo Fields**
```
Comment (jyokum) [inline README.md]: This section is no longer relevant and can be remove entirely.

**Review by jyokum — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mccrodp) [reply README.md]: Thanks @jyokum, missed that. Committed now.

**Pull request merged**